ts/pages/about/team.tsx

```diff
@@ -203,6 +203,16 @@ const team: TeamMember[] = [
         name: 'Fulvia Morales',
         title: 'Product Manager',
     },
+    {
+        imageUrl: '/images/team/kimp.jpg',
+        name: 'Kim Persson',
+        title: 'Front-end Engineer',
```

Comment:
Doesn't really matter but I think I'd prefer a more generic title 😄 
```suggestion
        title: 'Software Engineer',
```